指向和选择设定.md

```diff
@@ -0,0 +1,25 @@
+With OptiKey selected press 'ALT' and 'M' on your keyboard (physical or on-screen if using a tablet) to open the Management Console, then select the 'Pointing & Selecting' tab at the top. The Management Console must be accessed using a physical mouse and keyboard.
+
+在Optikey上，按下键盘的“ALT”和“M”按键（物理的或者平板屏幕上的）去打开管理控制台，然后在顶部选择“指向和选择”标签。必须使用物理鼠标和键盘才能访问管理控制台。
+
+![Image] (https://github.com/JuliusSweetland/OptiKey/raw/gh-pages/images/Management_Console_Pointing_And_Selecting_EyeTribe_Numbered.png)
+
+**Pointing with an eye tracker**
+**一个眼球跟踪设备的指示**
+
+1. Source: Which device is providing location data, e.g. the mouse, an eye tracker, etc. In this example the source is TheEyeTribe eye tracker.
+1. 来源：一个设备提供位置数据，比如鼠标、眼球跟踪设备等。这个例子中来源就是 TheEyeTribe 眼球跟踪设备。
+
+2. Time until point becomess irrelevent (ms): If the source device (e.g. an eye tracker) timestamps the location data, then this setting controls how old a received point can be before it is considered too old to be useful. These "stale" points will be logged for debugging purposes and then discarded.
+2. 点失效的时间（毫秒）：如果源设备（比如一个眼球跟踪设备）给位置数据打上时间戳，接下来这个设定控制一个接收点在失效前可以持续多久。调试过程中，这些“过时的”点可以被记录下来并被丢弃掉。
+
+**Selection with fixations**
+**固定选择**
+
+3. Key selection source: The method used to select a key. Possible methods are fixations, keyboard keys and mouse buttons. In this example the source is Fixations.
+3. 按键选择的来源：这个方法用来选择一个按键。可能的方法有固定、键盘按键和鼠标按键。这个例子中来源是固定的。
+
+4. Key fixation time to lock-on (ms): Specific to 'Key selection source' of 'Fixations', this setting dictates how long you must direct your attention to a key (using whatever 'Source' you have selected, e.g. by looking at a key when using an eye tracking source) before a fixation begins. The lock-on period is designed so that a deliberate pause is required on a key before a fixation begins. Without a 'lock-on' period every key will immediately begin a fixation as you direct your attention around the keyboard, which makes the interface busy and less clear.
+4. 锁定按键的固定时间：在“按键选择来源”是“固定”时，这个设定表明在固定开始前，你必须集中注意力多长时间在一个按键上（使用你选择的“来源”，比如当使用一个眼球跟踪设备时注视一个按键）。一个固定开始前，一个按键被故意设计成有一定延迟的锁定时间。如果没有一个锁定时间，当你集中注意力到键盘上，每个按键会立刻起作用，这样会让接口很忙而且不够清楚。
```

Comment:
首句遗漏 （ms），对于 ms 可直接引用或译为 毫秒。 

The lock-on period is designed so that a deliberate pause is required on a key before a fixation begins. 
断句应该是 [The lock-on period is designed] [so that] [a deliberate pause is required] [on] [ a key before a fixation begins. ] 
--->
在按键被固定选择之前，需要经过一个刻意设计的锁定延迟时间。